extensions/adobe/experience/campaign/experienceevent-all.schema.json

```diff
@@ -283,7 +283,7 @@
         "https://ns.adobe.com/experience/campaign/offerOpened": {
           "title": "Offer Opened",
           "description": "The offer in the message for an 'offer opened' event.",
-          "$ref": "https://ns.adobe.com/experience/offer-management/proposition-response-detail"
+          "$ref": "https://ns.adobe.com/experience/campaign/offer-response-detail"
```

Comment:
Is this going to be a breaking change to the existing schema and data?

Reply:
No, the Campaign team isn't able to generate data just yet. The needed the more specialized schema to finish their part. What this change does is to take a mere general schema offer-management/proposition-response-detail extend it to campaign/offer-response-detail and use the new, more specialized schema in the place where the old one was used.